src/js/jsx/shared/Datalist.jsx

```diff
@@ -78,12 +77,18 @@ define(function (require, exports, module) {
         },
 
         shouldComponentUpdate: function (nextProps, nextState) {
+            if (this.props.live) {
```

Comment:
With live Datalists, we have the value change as you hover over different options. I'll add in a check for `this.state.id !== nextState.id` (Before I added in the autosuggestions, `shouldCompontentUpdate` wasn't in Datalist at all)